app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { SafeAreaView } from 'react-native';
+
+import { IMAGES } from '../../../../assets';
+import {
+  BottomButtonView,
+  ExtendedText,
+  MainHeader,
+} from '../../../../components';
+import { useAppSelector } from '../../../../hooks';
+import { ISignUpSuccessScreenProps } from './SignUpSuccess.props';
+import { styles } from './SignUpSuccess.styles';
+
+export const SignUpSuccessScreen: React.FC<ISignUpSuccessScreenProps> = ({
+  navigation,
+  route,
+}) => {
+  const { t } = useTranslation();
+  const isChild = route.params?.isChild;
+  const avatar = useAppSelector(
+    state => state.cache.auth[isChild ? 'child' : 'parent']?.avatar,
+  );
+
+  const getCorrectLocalizationPath = () => {
+    if (isChild) {
+      return 'screens.onboarding.sign_up_success.child';
+    } else {
+      return 'screens.onboarding.sign_up_success.parent';
+    }
+  };
```

Comment:
`useMemo` and `[isChild]` in deps to update it correctly.
save the result to the variable and use it across the render